templates/build/news-feed-base.build.hbs

```diff
@@ -103,7 +103,7 @@
       \{{/ifCond}}
     \{{/ifCond}}
     \{{#ifCond limitEntries '>' 0}}
-    <p>Showing the first \{{limitEntries}} list items. \{{#if searchEnabled}}Search\{{/if}}\{{#ifCond searchEnabled '&&' filtersEnabled}} and/or \{{/ifCond}}\{{#if filtersEnabled}}Filter\{{/if}} to see more.</p>
+    <p>Showing the first \{{limitEntries}} \{{#ifCond searchEnabled '||' filtersEnabled}} list items. \{{#if searchEnabled}}Search\{{/if}}\{{#ifCond searchEnabled '&&' filtersEnabled}} and/or \{{/ifCond}}\{{#if filtersEnabled}}Filter\{{/if}} to see more. \{{else}} lists.\{{/ifCond}}</p>
```

Comment:
1. It looks like if neither search or filter is enabled, the message would say "Showing the first {{limitEntries}} lists.". Am I right? If so, it should be "Showing the first {{limitEntries}} list items."

Reply:
@tonytlwu Fix this.